site/content/docs/install.md

```diff
@@ -1,444 +1,115 @@
 ---
-title: "Install"
-date: 2017-01-04T15:04:10.000Z
+title: "SNAS Docker Install"
+date: 2017-05-07T15:04:10.000Z
 ---
 
-Install Steps
-=============
-See the various requirements and suggested system configurations at [Requirements](REQUIREMENTS.md)
+## SNAS Docker Install
 
-Install Using Docker
---------------------
+<!--more-->
 
-### docker hub: openbmp/aio
-All-in-one (aio) includes everything needed to run the collector and store the data in MySQL.   You can use this container to test/evaluate OpenBMP as well as run smaller deployments.  Production deployments normally would have distributed collectors and a redundant pair of MySQL/MariaDB servers. 
+(Prerequisite) Platform Docker Install
+--------------------------------------
 
-#### Container Includes
-* **Openbmpd** - Latest collector (listening port is TCP 5000)
-* **MariaDB 10.0** - MySQL server (listening port TCP 3306)
-* **Apache Kafka 0.8.x** - High performing message bus (listening ports are TCP 2181 and 9092)
-* **Tomcat/DB_REST** - Latest Rest interface into MySQL/MariaDB (listening port TCP 8001)
-* **Openbmp MySQL Consumer** - Latest Consumer that puts all data into MySQL
+> Ignore this step if you already have a current docker install. Go to [Install SNAS Using Docker](#install-snas-using-docker)
 
-### Recommended Current Linux Distributions
+> ####NOTE
```

Comment:
This isn't rendering correctly... Might need a space between NOTE and the last hash.